src/components/header/header.css

```diff
@@ -16,3 +16,7 @@
 .material-icons{
     font-size: 24px !important;
 }
+
+.menu {
+    right: -16px !important;
```

Comment:
This could be less (maybe `right: -8px !important;`) and the put the rest on `.wrapper`.
`.wrapper` has too much space on the right compared to nano 1.0.0 and not enough at the bottom. 

I mean something like this is missing
```
.wrapper {
  margin-right: -8px;
  margin-bottom: 16px;
}
```